x/ibc/02-client/keeper/keeper.go

```diff
@@ -147,6 +148,7 @@ func (k Keeper) GetSelfConsensusState(ctx sdk.Context, height uint64) (exported.
 // objects. For each State object, cb will be called. If the cb returns true,
 // the iterator will close and stop.
 func (k Keeper) IterateClients(ctx sdk.Context, cb func(exported.ClientState) bool) {
+	// TODO: determine how to fix this function with new key format
```

Comment:
Unsure of best way to fix this function with current key format that puts clientID before "client_state"

Reply:
This should only be called outside of the state machine (e.g. in queriers), so I think it's OK if it's not the most efficient.

Reply:
Hmm currently think clientstates are stored under `ibc/{clientID}/client_state` and consensus states are stored under `ibc/{clientID}/consensus_state/{height}`

Could add another client prefix in after ibc? then we would be iterating over client states and consensus states. would be somewhat inefficient but not terrible

Reply:
ah - it should be `ibc/clients/{clientID}/client_state` and `ibc/clients/{clientID}/consensus_state/{height}`, I think